pkg/proxy/proxy.go

```diff
@@ -269,10 +270,33 @@ func parseIPPort(ipstr string, info *accesslog.EndpointInfo) {
 	}
 }
 
-func (r *Redirect) getSourceInfo(req *http.Request) (accesslog.EndpointInfo, accesslog.IPVersion) {
+//   This function fills the accesslog.EndpointInfo fields, by fetching the consumable from the consumable cache
+//   of endpoint using identity sent by source. This is needed in ingress proxy while logging the source endpoint info.
+//   Since there will be 2 proxies on the same host, if both egress and ingress policies are set, the ingress policy cannot determine the
+//   source endpoint info based on ip address, as the ip address would be that of the egress proxy i.e host.
+
+func (r *Redirect) getInfoFromConsumable(ipstr string, info *accesslog.EndpointInfo, srcIdentity policy.NumericIdentity) {
+	ep := r.source
+	ip := net.ParseIP(ipstr)
+	if ip != nil {
+		if ip.To4() != nil {
+			info.IPv4 = ip.String()
+		} else {
+			info.IPv6 = ip.String()
+		}
+	}
+	secLabel := ep.ResolveIdentity(srcIdentity)
+
+	if secLabel != nil {
```

Comment:
Nit: can you rename this variable from 'secLabel' to something involving 'identity'? We want to move away from the 'secLabel' terminology.

Reply:
sure

Reply:
done. secLabel -> secIdentity